src/test/java/com/wikia/webdriver/elements/mercury/pages/login/RegisterPage.java

```diff
@@ -2,20 +2,21 @@
 
 import com.wikia.webdriver.common.core.elemnt.Wait;
 import com.wikia.webdriver.pageobjectsfactory.pageobject.WikiBasePageObject;
-import lombok.Getter;
+
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.WebElement;
 import org.openqa.selenium.support.FindBy;
 import org.openqa.selenium.support.PageFactory;
 
-public class JoinPage extends WikiBasePageObject {
+public class RegisterPage extends WikiBasePageObject {
 
     @FindBy(css = ".footer-callout-emphasis")
     private WebElement SignInButton;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this field "SignInButton" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS00116)